app/src/main/java/es/wolfi/passman/API/Vault.java

```diff
@@ -53,12 +66,24 @@
     public String public_sharing_key;
     public double last_access;
     public String challenge_password;
+    public int sharing_keys_generated;
+    public boolean delete_request_pending;
+    public JSONObject vault_settings = null;
+    public static Integer[] keyStrengths = {1024, 2048, 4096};
```

Comment:
On the phone, given it has a decent random source backend and decent cpu and your library of choice is probably using those, i would remove the 1024 length version, it's on the browser because key generation can be real slow (or at least at the time the system was implemented).

Reply:
I removed it. Feels also better with 2048 as minimum :)

Reply:
2048 is the default option at the moment

![grafik](https://user-images.githubusercontent.com/30630233/153072865-e187081b-e7ae-4966-99d8-58366a990f49.png)

I would not remove 1024. Maybe someone has a good reason to specially select it.